platforms/aws-graviton/prepare.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -eu
+
+cp ./code/package.json ./package.json
+
+if [ "$OSTYPE" = "linux" ]; then
```

Comment:
Why is this needed?

Reply:
Probably not required anymore, there was some command that I was using that only worked on `linux`. I will validate that and remove it.